Maker-Agents-Suit-Module/Governance-Module/src/lift.events.ts

```diff
@@ -0,0 +1,57 @@
+import { 
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType, 
+  Log,
+} from 'forta-agent';
+import { 
+  AddressVerifier,
+  isAddressKnown,
+  decodeSingleParam,
+} from './utils';
+
+const LIFT_EVENT: string = "0x3c278bd500000000000000000000000000000000000000000000000000000000";
+
+export const createFinding = (alertId: string, unknown: string, topic: number): Finding =>
+  Finding.fromObject({
+    name: "MakerDAO MCDM_ADM lift event detected",
+    description: `Topic #${topic} is an unknown address`,
+    alertId: alertId,
+    type: FindingType.Suspicious,
+    severity: FindingSeverity.High,
+    metadata: {
+      address: unknown,
+    },
+  });
+
+export const provideLiftEventsListener = (
+  alertId: string, 
+  contractAddress: string,
+  isKnown: AddressVerifier = isAddressKnown,
+  topic: string = LIFT_EVENT,
```

Comment:
Why do you need to receive the topic in the arguments, this handler is for detecting only `Lift` events, what is the point of receiving the topic as an argument?

Reply:
True, but I made the agent a little bit more general, because the `note` modifier is used in other functions and maybe later they ask for those events. Maybe I can move this implementation to a separate file, and make a new function that wraps it and pass the `LIFT_EVENT`. Or I can just deletes the topic argument as you suggest. What do you think?